client/src/app/components/Forms/Login/index.tsx

```diff
@@ -0,0 +1,124 @@
+    };
+    const { register, handleSubmit, formState: { errors } } = useForm({ resolver: yupResolver(loginUserValidation) })
+    const onSubmit = async (data: any) => {
+        try {
+            await loginUserValidation.isValidSync({ ...data }, { abortEarly: false });
+
+            const response = await fetch('api/login', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify(data)
+            });
+
+            const responseData = await response.json();
+
+
+            if (responseData.status === RESULT_STATUS.SUCCESS) {
+
+                const token = responseData.data.token;
+                document.cookie = `token=${token}; path=/`;
+                router.push('/task')
+
+                return NextResponse.json(data);
+            }
+            if (responseData.status === RESULT_STATUS.FAILURE) {
+                setErrorMessage(responseData.message);
+            }
+
+        } catch (error) {
+            console.error('Error while login:', error);
+            setErrorMessage("An error occurred during login. Please try again later.");
+            return NextResponse.error()
+
+        }
+    }
+
+
+
+
+    return (
+        <div>
+
+            <form
+                onSubmit={handleSubmit(onSubmit)}
+                className="flex flex-col align-middle bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 max-w-md">
+
+                <div className="mb-4">
+                    <label className="block text-gray-700 text-sm font-bold mb-2" htmlFor="email">
+                        Email*
+                    </label>
+                    <input
+                        {...register('email')}
+                        className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                        id="email"
+                        type="email"
+                        placeholder="Email"
+                    />
+                    <span className="text-red-500">{errors.email?.message}</span>
+                </div>
+                <div className="mb-6">
+                    <label className="block text-gray-700 text-sm font-bold mb-2" htmlFor="password">
+                        Password*
+                    </label>
+                    <div className="relative">
+                        <input
+                            {...register('password')}
+                            className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 mb-3 leading-tight focus:outline-none focus:shadow-outline"
+                            id="password"
+                            type={showPassword ? 'text' : 'password'}
+                            placeholder="******************"
+                        />
+                        <span className="absolute inset-y-0 right-0 flex items-center pr-3">
```

Comment:
User grid for creating UI of form [Grid](https://tailwindcss.com/docs/grid-template-columns)